services/ui-src/src/components/banners/AdminBannerProvider.tsx

```diff
@@ -41,7 +41,10 @@ export const AdminBannerProvider = ({ children }: Props) => {
       const currentBanner = await getBanner(ADMIN_BANNER_ID);
       const newBannerData = currentBanner?.Item || {};
       setBannerData(newBannerData);
-      setBannerErrorMessage("");
+      setBannerErrorMessage({
```

Comment:
I still see a persistent banner here. Since we are deliberately calling `setBannerErrorMessage` in the try block with error verbiage, the alert is going to show up when we're checking the store. This should probably be `setBannerErrorMessage(undefined)`